acestep/customized_vllm/__init__.py

```
@@ -0,0 +1,393 @@
+
+    # -- Internal generation logic ----------------------------------------
+
+    def _prepare_slots(self, prompts, sampling_params, unconditional_prompts):
+        """Tokenise prompts, create slots, allocate KV cache."""
+        if not isinstance(sampling_params, list):
+            sampling_params = [sampling_params] * len(prompts)
+        if unconditional_prompts is None:
+            unconditional_prompts = [None] * len(prompts)
+
+        all_slots = []
+        for prompt, sp, uncond in zip(prompts, sampling_params, unconditional_prompts):
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)